vowpalwabbit/distributionally_robust.h

```diff
@@ -139,63 +134,11 @@ namespace distributionally_robust
       return duals.second.qfunc(w, r);
     }
 
-    void save_load(io_buf& model_file, bool read, bool text)
```

Comment:
Where is this function being called? You are going to need to ensure the original callsite is using `VW::model_utils::process_model_field`

Reply:
This isnt being called anywhere in master, but it's being used (and tested) in the automl branch.